setup.py

```diff
@@ -30,8 +30,6 @@ def get_requirements() -> Sequence[str]:
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     extras_require={
-        "avro": ["avro>=1.11.1"],
         "json": ["python-rapidjson>=1.8", "fastjsonschema>=2.16.2"],
```

Comment:
Do we wanna remove this too?

Reply:
but commit log codec needs it, right?

Reply:
ah oops indeed it just uses the Codec class. should we remove that one aswell?

Reply:
Yeah I think we should since it's in sentry-kafka-schemas (IIRC it's a different one to the commit log codec)